File: dataset/output/MISTRAL_codestral-latest/Task15_MISTRAL_codestral-latest.java
import java.util.ArrayList;
import java.util.List;

public class TowerBuilder {
    public static List<String> buildTower(int nFloors) {
        List<String> tower = new ArrayList<>();
        for (int i = 0; i < nFloors; i++) {
            String floor = " ".repeat(nFloors - i - 1) + "*".repeat(2 * i + 1) + " ".repeat(nFloors - i - 1);
            tower.add(floor);
        }
        return tower;
    }
}

File: dataset/output/MISTRAL_codestral-latest/Task15_MISTRAL_codestral-latest.cpp
#include <vector>
#include <string>

std::vector<std::string> towerBuilder(int nFloors) {
    std::vector<std::string> tower;
    for (int i = 0; i < nFloors; i++) {
        tower.push_back(std::string(nFloors - i - 1, ' ') + std::string(2 * i + 1, '*') + std::string(nFloors - i - 1, ' '));
    }
    return tower;
}